### PR TITLE
Go-Pro module

### PR DESCRIPTION
**Pull request by arthurbenemann**

### CODE REVIEW

File: MAVProxy/modules/mavproxy_gopro.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+'''arm/disarm command handling'''
```
Comment (tridge) [inline MAVProxy/modules/mavproxy_gopro.py]: hi Arthur, can you update the comments in the top of the module to explain what it does and how you need to setup the gopro to use it?

Comment (arthurbenemann) [inline MAVProxy/modules/mavproxy_gopro.py]: Done

**Comment by arthurbenemann** [comment]
@tridge it would be nice if you merged this, I hope to improve the module soon. But the messages are likely going to change in the near future.

**Comment by tridge** [comment]
applied, thanks!

**Pull request closed without merging**